doc/source/tutorial/ndimage.rst

```diff
@@ -1,5 +1,5 @@
-Multidimensional image processing (`scipy.ndimage`)
-====================================================
+Multidimensional Image Processing (`scipy.ndimage`)
```

Comment:
Just as a point of reference, NumPy went the other direction w.r.t. capitalization (gh-25016).